package.json

```
@@ -1,64 +1,39 @@
-    "dev": "tsc-watch --preserveWatchOutput --noClear",
-    "dev-server": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/server && npm run dev\"",
-    "dev-script": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/script && npm run dev\"",
-    "dev-bundle": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/bundle && npm run dev\"",
-    "dev-ios": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/native && npm run dev-ios\"",
-    "dev-android": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/native && npm run dev-android\"",
-    "dev-test-helper": "cd examples/test-helper && npm run dev",
-    "test": "npm run lint && npm run licenses && npm run test-unit",
-    "test-unit": "jest --coverage --verbose",
-    "test-acceptance": "npm run test-acceptance-server && npm run test-acceptance-script && npm run test-acceptance-bundle && npm run test-acceptance-ios && npm run test-acceptance-android",
-    "test-acceptance-server": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-script": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-bundle": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-ios": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-android": "echo \"Not Implemented\" && exit 1",
-    "bootstrap-examples": "./bootstrap-examples.bash",
-    "build-api-docs": "npx typedoc --out docs/api/source/api",
-    "build-docs-preview-site": "npm run build-api-docs; cd docs/prose; make html; cd ../api; make html; cd ../; rm -r dist || true; mkdir -p dist/prose dist/api; cp -r prose/build/html/. dist/prose/; cp -r api/build/html/. dist/api/; echo 'Draft documentation: <a href=\"./prose/\">Prose</a>, <a href=\"./api/\">API docs</a>.' >> dist/index.html"
+    "format-all": "prettier --write \"packages/*/src/**\" \"packages/*/__tests__/**\"",
+    "jest-all": "jest --coverage --verbose",
+    "licenses-all": "license-checker --production --failOn \"AGPL-1.0-only; AGPL-1.0-or-later; AGPL-3.0-only; AGPL-3.0-or-later; Beerware; CC-BY-NC-1.0; CC-BY-NC-2.0; CC-BY-NC-2.5; CC-BY-NC-3.0; CC-BY-NC-4.0; CC-BY-NC-ND-1.0; CC-BY-NC-ND-2.0; CC-BY-NC-ND-2.5; CC-BY-NC-ND-3.0; CC-BY-NC-ND-4.0; CC-BY-NC-SA-1.0; CC-BY-NC-SA-2.0; CC-BY-NC-SA-2.5; CC-BY-NC-SA-3.0; CC-BY-NC-SA-4.0; CPAL-1.0; EUPL-1.0; EUPL-1.1; EUPL-1.1;  GPL-1.0-only; GPL-1.0-or-later; GPL-2.0-only;  GPL-2.0-or-later; GPL-3.0; GPL-3.0-only; GPL-3.0-or-later; SISSL;  SISSL-1.2; WTFPL\"",
+    "lint-all": "eslint \"packages/*/src/**/*.ts\" && eslint \"packages/*/__tests__/**/*.ts\"",
+    "test-all": "npm run lint-all && npm run licenses-all && npm run test-unit-all",
+    "test-unit-all": "jest --coverage --verbose",
+    "test": "lerna run build",
```

I think you meant this?

```suggestion
    "test": "lerna run test",
```